palomabwizeline/react-apprenticeship-capstone4
# Feat/deliverable1

## Description
```
Deliverable 1 for Capstone Project 4.

```
Delete options that are not relevant.

```


## How should this be tested?
https://wizemarket.netlify.app/

```
Add a brief description of how can be tested this deliverable step by step. 
I know styles look better on mobile than on desktop, I need to work on responsive design.

```

Add any information regarding the PR that the reviewers should know, if necessary, for example "This is only a fix PR related the past PR (PR link)" 
Test is not working because my logo/name is a Link and apparently is not allowed to be there. I will try to fix it.

```
## Screenshots and links
![Captura de pantalla 2022-02-07 a las 10 36 07](https://user-images.githubusercontent.com/97017377/152762513-f3ba7ba5-e6f5-43d9-b492-2e411bc65452.png)

```
